data/npc/scripts/bank.lua

```diff
@@ -199,6 +199,10 @@ local function creatureSayCallback(cid, type, msg)
 			receiver = receiver:trim()
 
 			-- Immediate topicList.TRANSFER_PLAYER_GOLD simulation
+			if tonumber(parts[3]) then
```

Comment:
why not try and swap them if parts[2] is not a number?

Reply:
sure makes sense actually